commands/confirm.py

```diff
@@ -12,26 +13,34 @@
 uw_driver = UW_Driver()
```

Comment:
We ca get rid of this right?

Reply:
Yeah we can probably get rid of all of the uw driver stuff now.